pkg/workers/exec/service.go

```diff
@@ -50,6 +50,9 @@ func (w *serviceWorker) PrepareWorkDir(ctx *jobs.WorkerContext, i *instance.Inst
 		}
 		return
 	}
+
+	w.slug = slug
```

Comment:
By curiosity, why did you move this assignation?

Reply:
Only for the logs. If we get the manifest but have any kind of error after (app not ready, service not found, ....), we will still have the slug name instead of an empty one